tensorflow_probability/python/distributions/ordered_logistic_test.py

```diff
@@ -0,0 +1,156 @@
+# Copyright 2020 The TensorFlow Probability Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ============================================================================
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+# Dependency imports
+import numpy as np
+import tensorflow.compat.v2 as tf
+import tensorflow_probability as tfp
+
+from tensorflow_probability.python.internal import test_util
+
+tfd = tfp.distributions
+
+
+@test_util.test_all_tf_execution_regimes
+class OrderedLogisticTest(test_util.TestCase):
+
+  def setUp(self):
+    self._rng = np.random.RandomState(123)
+    super(OrderedLogisticTest, self).setUp()
+
+  def testBatchShapes(self):
+
+    for test in ["cutpoints", "location", "both"]:
```

Comment:
I would recommend using the @parameterized decorator when creating tests with for loops:
https://github.com/tensorflow/probability/blob/c83172e7612d454825aeca2571e4fdde38fcb863/tensorflow_probability/python/internal/nest_util_test.py#L68

Here and below

Reply:
Cool, I've not seen that before. Done in https://github.com/tensorflow/probability/pull/753/commits/25e1e0d5332f53ba77f1c3dd8d8a07e491be9941